sdk/keyvault/azure-security-test-keyvault-jca/src/test/java/com/azure/security/keyvault/jca/test/PropertyConvertorUtils.java

```diff
@@ -31,7 +31,8 @@ public static void putEnvironmentPropertyToSystemPropertyForKeyVaultJca() {
     public static final List<String> KEYVAULT_JCA_SYSTEM_PROPERTIES = Arrays.asList("AZURE_KEYVAULT_ENDPOINT",
         "AZURE_KEYVAULT_TENANT_ID",
         "AZURE_KEYVAULT_CLIENT_ID",
-        "AZURE_KEYVAULT_CLIENT_SECRET");
+        "AZURE_KEYVAULT_CLIENT_SECRET",
+        "CERTIFICATE_AZURE_KEYVAULT_URI"); //Don't sure if AZURE_KEYVAULT_URI will affect other module, so add prefix
```

Comment:
`//Don't` -> `// Not`.

Reply:
If we add prefix for `AZURE_KEYVAULT_URI`, do we need to add prefix for other variables like `AZURE_KEYVAULT_TENANT_ID` /  `AZURE_KEYVAULT_CLIENT_ID`. How about deleting the prefix or add the prefix for all variables?